content/docs/configuration/acme/dns01/route53.md

```diff
@@ -47,30 +47,382 @@ permissions:
+
+#### EKS Pod Identity
+
+[EKS Pod Identity](https://docs.aws.amazon.com/eks/latest/userguide/pod-identities.html) is the simplest way to use ambient credentials,
+if you deploy cert-manager on EKS.
+It is a four step process:
+1. [Setup the EKS Pod Identity agent](https://docs.aws.amazon.com/eks/latest/userguide/pod-id-agent-setup.html) in your cluster.
+2. [Assign an IAM role to the cert-manager Kubernetes service account](https://docs.aws.amazon.com/eks/latest/userguide/pod-id-association.html).
+3. Restart the cert-manager Deployment
+   so that the EKS Pod Identity Agent can inject the necessary environment variables into the Pods.
+4. Create a `ClusterIssuer` resource:
+
+   ```yaml
+   apiVersion: cert-manager.io/v1
+   kind: ClusterIssuer
+   metadata:
+     name: letsencrypt-prod
+   spec:
+     acme:
+       solvers:
+       - dns01:
+           route53: {}
+   ```
+
+#### EKS IAM Role for Service Accounts (IRSA)
+
+IAM Roles for Service Accounts (IRSA) is another way to use ambient credentials,
+if you deploy cert-manager on EKS.
+It is more complicated than Pod Identity and requires coordination between the Kubernetes cluster administrator and the AWS account manager.
+It involves annotating the `cert-manager` ServiceAccount in Kubernetes, and setting up an IAM role, a trust policy and a trust relationship in AWS.
+A mutating webhook will automatically setup a mounted service account volume in the cert-manager Pod.
+
+1. **Create an IAM OIDC provider for your cluster**
+
+   To use IRSA with cert-manager you must first enable the feature for your cluster.
+   Follow the [official documentation](https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html).
+
+2. **Create a trust relationship**
+
+   In this configuration an IAM role is mapped to the cert-manager `ServiceAccount` allowing it to authenticate with AWS.
+   The IAM role you map to the `ServiceAccount` will need permissions on any and all Route53 zones cert-manager will be using.
+   Create a trust relationship by adding the following trust policy to the IAM role:
+
+   ```json
+   {
+     "Version": "2012-10-17",
+     "Statement": [
+       {
+         "Effect": "Allow",
+         "Action": "sts:AssumeRoleWithWebIdentity",
+         "Principal": {
+           "Federated": "arn:aws:iam::<aws-account-id>:oidc-provider/oidc.eks.<aws-region>.amazonaws.com/id/<eks-hash>"
+         },
+         "Condition": {
+           "StringEquals": {
+             "oidc.eks.<aws-region>.amazonaws.com/id/<eks-hash>:sub": "system:serviceaccount:<namespace>:<service-account-name>"
+           }
+         }
+       }
+     ]
+   }
+   ```
+
+   Replace the following:
+
+   - `<aws-account-id>` with the AWS account ID of the EKS cluster.
+   - `<aws-region>` with the region where the EKS cluster is located.
+   - `<eks-hash>` with the hash in the EKS API URL; this will be a random 32 character hex string (example: `45DABD88EEE3A227AF0FA468BE4EF0B5`)
```

Comment:
nitpick (non-blocking): All the rest of the bullets here end in a period but the `eks-hash` one doesn't